src/main/scala/com/github/unchama/seichiassist/subsystems/gacha/System.scala

```diff
@@ -0,0 +1,54 @@
+package com.github.unchama.seichiassist.subsystems.gacha
+
+import cats.effect.ConcurrentEffect.ops.toAllConcurrentEffectOps
+import cats.effect.{ConcurrentEffect, Sync, SyncIO}
+import com.github.unchama.concurrent.NonServerThreadContextShift
+import com.github.unchama.minecraft.actions.OnMinecraftServerThread
+import com.github.unchama.seichiassist.meta.subsystem.Subsystem
+import com.github.unchama.seichiassist.subsystems.gacha.bukkit.actions.BukkitDrawGacha
+import com.github.unchama.seichiassist.subsystems.gacha.bukkit.command.GachaCommand
+import com.github.unchama.seichiassist.subsystems.gacha.bukkit.listeners.GachaController
+import com.github.unchama.seichiassist.subsystems.gacha.domain.GachaPrizesDataOperations
+import com.github.unchama.seichiassist.subsystems.gacha.domain.bukkit.GachaPrize
+import com.github.unchama.seichiassist.subsystems.gacha.infrastructure.bukkit.JdbcGachaPersistence
+import com.github.unchama.seichiassist.subsystems.gacha.subsystems.gachaticket.infrastructure.JdbcGachaTicketPersistence
+import com.github.unchama.seichiassist.subsystems.itemmigration.domain.minecraft.UuidRepository
+import org.bukkit.command.TabExecutor
+import org.bukkit.entity.Player
+import org.bukkit.event.Listener
+
+trait System[F[_]] extends Subsystem[F] {
+  val api: GachaReadAPI[F, Player] with GachaWriteAPI[F]
+}
+
+object System {
+
+  def wired[F[_]: OnMinecraftServerThread: NonServerThreadContextShift: ConcurrentEffect: Sync](
```

Comment:
```suggestion
  def wired[F[_]: OnMinecraftServerThread: NonServerThreadContextShift: ConcurrentEffect](
```

`ConcurrentEffect`は自明に`Sync`です (https://typelevel.org/cats-effect/docs/2.x/typeclasses/overview 参照)